src/link.ts

```diff
@@ -0,0 +1,88 @@
+import { html, LitElement } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { createRef, ref } from 'lit/directives/ref.js';
+import packageJson from '../package.json' with { type: 'json' };
+import styles from './link.styles.js';
+import shadowRootMode from './library/shadow-root-mode.js';
+import final from './library/final.js';
+import required from './library/required.js';
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'glide-core-link': GlideCoreLink;
+  }
+}
+
+/**
+ * @attr {string} label
+ * @attr {string} url
+ * @attr {boolean} [disabled=false]
+ * @attr {string} [download]
+ * @attr {'_blank'|'_parent'|'_self'|'_top'} [target]
+ *
+ * @readonly
+ * @attr {string} [version]
+ */
+@customElement('glide-core-link')
+@final
+export default class GlideCoreLink extends LitElement {
+  static formAssociated = true;
+
+  static override shadowRootOptions: ShadowRootInit = {
+    ...LitElement.shadowRootOptions,
+    delegatesFocus: true,
+    mode: shadowRootMode,
+  };
+
+  static override styles = styles;
+
+  @property({ reflect: true })
+  @required
+  label?: string;
+
+  @property({ reflect: true })
+  @required
+  url?: string;
```

Comment:
Could we instead call this `href`, to match native `<a>` tags?

Reply:
> Could we instead call this href, to match native <a> tags?

I debated it. But I thought that `url` is more natural. And `href` implies that Link uses an `<a>`. It does currently. But it may not, and we should be able to swap the `<a>` out internally for something else and Link shouldn't have to change its API.

`url` instead of `href` is also a small thing we can do separate in people's minds that not every instance of `<a>` should be a Link. Much like not every instance of `<button>` should be a Button.

Reply:
Or is everything but my last point at odds with the fact that Link has `download` and `target` attributes?

Reply:
Ha, I guess so :) 

I don't feel too strongly either way

Reply:
Hah. Well, I'm in the `href` camp now for that reason. But I'll let @ynotdraw chime in next week.

Reply:
Sounds good! `href` it is.